bin/www

```diff
@@ -14,15 +16,29 @@ var RTM_CLIENT_EVENTS = require('@slack/client').CLIENT_EVENTS.RTM;
 var channels = require('../lib/constants/channels').MyChannels;
 
 var token = process.env.BUZZKILLFEED_SLACKBOT_TOKEN || '';
+var channelNames = (process.env.CHANNEL_NAMES || 'blurp').split(",");
 var ClickbaitGeneratorUrl = process.env.CLICKBAIT_GENERATOR_URL || '';
 
 var rtm = new RtmClient(token, {logLevel: 'warn'});
-var myId = 'U11N8LMAM';
+var myId;
+var myName;
+let channelIds = [];
 
+console.log("Starting with ", token, channelNames);
 rtm.start();
 
+
 rtm.on(CLIENT_EVENTS.RTM.AUTHENTICATED, function (rtmStartData) {
-    myId = rtmStartData['self']['id'];
+  myId = rtmStartData.self.id;
+  myName = rtmStartData.self.name;
+  console.log(`Logged in as ${myName} of team ${rtmStartData.team.name}, but not yet connected to a channel`);
+
+  for (const c of rtmStartData.channels) {
+    if (c.is_member && _.includes(channelNames, c.name)) {
```

Comment:
it occurs to me here that we could just act on all channels we've been invited to... so channel name could kind of go out the window.

unless we're trying to be polite.

i'm not sure how often this AUTHENTICATED event happens but if we get invited to a new channel after the app has started, are we going to capture that?  something to think about/test/check.